learning_resources/etl/sloan.py

```diff
@@ -0,0 +1,219 @@
+"""ETL for Sloan Executive Education data."""
+
+import logging
+from datetime import UTC
+from urllib.parse import urljoin
+
+import requests
+from dateparser import parse
+from django.conf import settings
+
+from learning_resources.constants import (
+    CertificationType,
+    OfferedBy,
+    PlatformType,
+    RunAvailability,
+)
+from learning_resources.etl.constants import ETLSource
+from learning_resources.etl.utils import (
+    transform_format,
+    transform_topics,
+)
+
+log = logging.getLogger(__name__)
+
+OFFERED_BY_CODE = OfferedBy.see.name
+
+
+def _get_access_token():
+    """
+    Get an access token for edx
+
+    Args:
+        config (OpenEdxConfiguration): configuration for the openedx backend
+
+    Returns:
+        str: the access token
+    """
+    payload = {
+        "grant_type": "client_credentials",
+        "client_id": settings.SEE_API_CLIENT_ID,
+        "client_secret": settings.SEE_API_CLIENT_SECRET,
+        "token_type": "jwt",
+    }
+    response = requests.post(  # noqa: S113
+        settings.SEE_API_ACCESS_TOKEN_URL, data=payload
+    )
+    response.raise_for_status()
+
+    return response.json()["access_token"]
+
+
+def parse_topics(topic):
+    """
+    Parse topics from a string
+
+    Args:
+        topic (str): the topic string
+
+    Returns:
+        list: the parsed topics
+    """
+
+    return (
+        transform_topics([{"name": topic.split(":")[-1].strip()}], OFFERED_BY_CODE)
+        if topic
+        else []
+    )
+
+
+def parse_image(course_data):
+    """
+    Parse image from course data
+
+    Args:
+        course_data (dict): the course data
+
+    Returns:
+        str: the image URL
+    """
+    image_url = course_data.get("Image_Src")
+    if image_url:
+        return {
+            "url": image_url,
+            "alt": course_data.get("Title"),
+            "description": course_data.get("Title"),
+        }
+    return None
+
+
+def parse_datetime(value):
+    """
+    Parses a datetime string
+
+    Args:
+        value(str): the datetime in string format
+
+    Returns:
+        datetime: the parsed datetime
+    """  # noqa: D401
+    return parse(value).replace(tzinfo=UTC) if value else None

```

Comment:
This might need to be adjusted a bit, it is showing dates a day earlier than expected (example, a course starting on 10/29/2024 is shown with a start date of 10/28/2024).  Maybe try something like this:

```python
from datetime import UTC
from zoneinfo import ZoneInfo

return parse(value).replace(tzinfo=ZoneInfo("US/Eastern")).astimezone(UTC)
```